### PR TITLE
Fix filebeat daemonset template

### PR DESCRIPTION
**Pull request by atsikham**
Task #2233 

Checked with `any` provider.

### CODE REVIEW

File: docs/home/howto/LOGGING.md [hunk 1]
```diff
@@ -259,7 +259,7 @@ Elasticsearch. For non-production environment you can easily disable it by addin
 
 ---
 
-## How to add multiline support for Filebeat logs
+## Multiline support for Filebeat logs
```
Comment (rafzei) [inline docs/home/howto/LOGGING.md]: If you want to change this header you have change reference here: https://github.com/epiphany-platform/epiphany/blob/develop/docs/home/HOWTO.md

File: docs/home/howto/LOGGING.md [hunk 2]
```diff
@@ -210,7 +210,7 @@ PUT _template/template_01
 After applying this policy, every new index created under this one will apply to it. There is also possibility to apply
 policy to already existing policies by assigning them to policy in Index Management Kibana panel.
 
-## How to export Elasticsearch data to csv format
+## Export Elasticsearch data to CSV format
```
Comment (rafzei) [inline docs/home/howto/LOGGING.md]: If you want to change this header you have change reference here: https://github.com/epiphany-platform/epiphany/blob/develop/docs/home/HOWTO.md

File: docs/home/howto/LOGGING.md [hunk 3]
```diff
@@ -56,7 +56,7 @@ specification:
     logs: /var/log/elasticsearch
 ```
 
-## How to manage Opendistro for Elasticsearch data
+## Opendistro for Elasticsearch data management
```
Comment (rafzei) [inline docs/home/howto/LOGGING.md]: If you want to change this header you have change reference here: https://github.com/epiphany-platform/epiphany/blob/develop/docs/home/HOWTO.md

**Review by rafzei — CHANGES_REQUESTED** [review]
What about changelog?
EDIT: maybe its worth to add "Filebeat as Daemonset in K8s" to HOWTO.md as well

**Review by mkyc — APPROVED** [review]

**Review by rafzei — APPROVED** [review]

**Pull request merged**